src/reuse/init.py

```diff
@@ -125,8 +125,13 @@ def run(args, project: Project, out=sys.stdout):
         except URLError:
             out.write(_("Could not download {}").format(lic))
             out.write("\n")
-        except FileNotFoundError:
-            out.write(_("Could not copy {}").format(lic))
+        except FileNotFoundError as err:
+            out.write(
+                _(
+                    "Error: Could not copy {path}, "
+                    "please add manually {lic}.txt in the LICENCES/ directory."
```

Comment:
Grammar: `please add {lic}.txt manually in the LICENSES directory.`

Reply:
And it does sound wrong when re-reading it. Fixed !